oracle-db/info.json

```diff
@@ -1,84 +1,85 @@
-                "editable": true,
-                "description": "Specify the username to access the database to which you will connect and perform automated operations."
-            },
-            {
-                "name": "password",
-                "required": false,
-                "type": "password",
-                "visible": true,
-                "title": "Password",
-                "editable": true,
-                "description": "Specify the password to access the database to which you will connect and perform automated operations."
-            }
-        ]
-    },
-    "operations": [
+  "name": "oracle-db",
+  "version": "1.0.0",
+  "label": "Oracle Database",
+  "description": "Use Oracle Database connector to connect to a database and then query the database and retrieve data. supports Oracle v12c onward. ",
+  "publisher": "Fortinet CSE",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "category": "utilities",
+  "help_online": "https://docs.fortinet.com/document/fortisoar/1.0.0/oracle-database/463/oracle-database-v1-0-0",
+  "configuration": {
+    "fields": [
+      {
+        "name": "host",
+        "required": true,
+        "type": "text",
+        "visible": true,
+        "title": "Host",
+        "editable": true,
+        "description": "Specify the Hostname of the database server to which you will connect and perform the automated operations."
+      },
+      {
+        "name": "port",
+        "required": true,
+        "type": "text",
+        "visible": true,
+        "title": "Port",
+        "editable": true,
+        "description": "Specify the name of the database to which you will connect and perform name automated operations."
+      },
+      {
+        "name": "database",
+        "required": true,
+        "type": "text",
+        "visible": true,
+        "title": "Database name",
+        "editable": true,
+        "description": "Specify the name of the database to which you will connect and perform automated operations."
+      },
+      {
+        "name": "username",
+        "required": false,
+        "type": "text",
+        "visible": true,
+        "title": "Username",
+        "editable": true,
+        "description": "Specify the username to access the database to which you will connect and perform automated operations."
+      },
+      {
+        "name": "password",
+        "required": false,
+        "type": "password",
+        "visible": true,
+        "title": "Password",
+        "editable": true,
+        "description": "Specify the password to access the database to which you will connect and perform automated operations."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "db_query",
+      "annotation": "db_query",
+      "title": "Query DB",
+      "description": "Performs a query on the configured database based on the query string you have specified.",
+      "enabled": true,
+      "output_schema": {},
```

Comment:
Output Schema is missing

Reply:
This action execute DB query and DB query response not same its change depends on query. query like (insert, delete, select, update etc.)